tests/config/prettier-plugins/prettier-plugin-dummy-stylus/index.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+const parserName = "stylus-dummy-parser";
+const astName = parserName + "-ast";
+
+module.exports = {
+  languages: [
+    {
+      name: "stylus",
+      parsers: [parserName],
+    },
+  ],
+  parsers: {
+    [parserName]: {
+      parse: (text) => ({ text }),
+      astFormat: astName,
+      locStart() {},
+      locEnd() {},
+    },
+  },
+  printers: {
+    [astName]: {
+      print(path) {
+        return path.getValue().text + "\n// by dummy parser";
```

Comment:
I think this has to be a fixed string, second run will add one more comment , I guess the CI will fail.